src/coordination/coordinator_instance.cpp

```diff
@@ -188,9 +197,9 @@ auto CoordinatorInstance::GetLeaderCoordinatorData() const -> std::optional<Coor
   return raft_state_->GetLeaderCoordinatorData();
 }
 
-auto CoordinatorInstance::ShowInstances() const -> std::vector<InstanceStatus> {
+auto CoordinatorInstance::GetCoordinatorsInstanceStatus() const -> std::vector<InstanceStatus> {
   auto const stringify_coord_health = [this](CoordinatorToCoordinatorConfig const &instance) -> std::string {
-    if (!raft_state_->IsLeader()) {
+    if (!is_leader_ready_) {
```

Comment:
what does it mean that leader isn't ready?

Reply:
Leader is ready when it started all the instances and when it did force reset. We can't behave as leader until that is done

Reply:
okay, I was confused with is_leader_ready_. If this is always false on follower then we should rename it to something like if_i_am_ready_leader -> something like this not exactly. becaue is_leader_ready_ sounds to me like a state that can be read from nuraft logs

Reply:
We can update it in next PR